presto-main/src/main/java/io/prestosql/execution/TaskStatus.java

```diff
@@ -72,6 +75,8 @@
 
     private final List<ExecutionFailureInfo> failures;
 
+    private final Map<String, TupleDomain<String>> dynamicTupleDomains;
```

Comment:
I think we could just have `Map<String, Domain>`

Reply:
Done